releasenotes/notes/add-textbook-circuit-style-98600038608c8f75.yaml

```
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Added a new circuit drawing style, ``textbook``, which uses the colour
```

```suggestion
    Added a new circuit drawing style, ``textbook``, which uses the color
```

(as much as it pains me to make that suggestion...)

---

Ahh yes thank you